corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -143,14 +144,28 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
         $('#select-case').off('click').click(function () {
             FormplayerFrontend.trigger("menu:select", caseId);
         });
+        $('#select-case-for-multi-select').off('click').click(function () {
+            console.log("case selected!");
+            // select checkbox for case
+        });
         $('#case-detail-modal').find('.js-detail-tabs').html(tabListView.render().el);
         $('#case-detail-modal').find('.js-detail-content').html(menuListView.render().el);
         $('#case-detail-modal').modal('show');
 
         if (model.isPersistentDetail) {
+            $('#case-detail-modal').find('#select-case-for-multi-select').hide();
```

Comment:
You're following the existing pattern here, which makes sense, but the existing pattern is a kind of weird jQuery-instead-of-Marionette way to display elements, and you're adding enough logic that I'd argue it's time to graduate this code to become a real view.

I think this would be similar to the patterns used in this code for other views: 
* Extract the [footer HTML](https://github.com/dimagi/commcare-hq/blob/5d686f04affc14ed6b8eba8b175a745e8896d444/corehq/apps/cloudcare/templates/formplayer/case_detail.html#L44-L57) (or make a new div inside the footer HTML) into a template in that same file, the way `detail-view-tab-list-template` is extracted.
* Create a view similar to `DetailTabView`, and pass `isPersistentDetail` and `isMultiSelect` to it as options. That view would control the events for the buttons, which would be more Marionette-y than the pre-existing [off/click logic](https://github.com/dimagi/commcare-hq/blob/5d686f04affc14ed6b8eba8b175a745e8896d444/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js#L144-L150).
* Construct and render the new view above, similar to how `tabListView` is constructed and rendered.

Reply:
[2c15661](https://github.com/dimagi/commcare-hq/pull/31349/commits/2c15661474e6eb89e4a6b68062a32a468335df89)